src/jsifier.js

```diff
@@ -144,7 +158,7 @@ function JSify(data, functionsOnly) {
       functionStubSigs[item.ident] = Functions.getSignature(item.returnType.text, item.params.map(function(arg) { return arg.type }), false);
     }
 
-    function addFromLibrary(ident) {
+    function addFromLibrary(ident, firstCall) {
```

Comment:
Why does this need this `firstCall` boolean? This looks somewhat arbitrary?

Reply:
I guess it looks odd, yeah. I'll make it more explicit, give addFromLibrary a parameter "export".